src/controllers/contest.controller.js

```diff
@@ -0,0 +1,9 @@
+const Contest = require('../models/contest.model');
+
+exports.status = async (req, res, next) => {
+  try {
+    return res.json({ message: 'Contest endpoint is working.'})
```

Comment:
use callback error handlers for JS instead of try catch

Reply:
Try catch is usually fine, express can also automatically handle errors thrown, so there is no need for explicit calling of `next`.

When we are dealing with async functions however we might need a defined rule when it comes to handling promise resolutions (`await` or `promise.then()`).

Depending on these different ways we can either use try catch or `promise.catch()`.

https://expressjs.com/en/guide/error-handling.html

Reply:
Going to go with Yong here, he has express expertise. Updating the linting and CI process in another PR.